geotrek/tourism/filters.py

```diff
@@ -32,3 +32,11 @@ class Meta(StructureRelatedFilterSet.Meta):
         model = TouristicEvent
         fields = StructureRelatedFilterSet.Meta.fields + [
             'published', 'type', 'themes', 'after', 'before', 'approved']
+
+
+class TouristicEventApiFilterSet(django_filters.FilterSet):
+    ends_after = django_filters.DateFilter('end_date', lookup_type='gte')
+
+    class Meta:
+        model = TouristicEvent
+        fields = ['end_date', ]
```

Comment:
Not exactly, I would suggest a test where you create a few records and check that when you call the view with the filter you don't obtain every of them.
Think the tests as a specification of what you implemented.
Like if you removed or change this new code, some test should fail somewhere.
It really helps for long term maintenance!